docs/TPM_with_LPC_protocol.md

```diff
@@ -0,0 +1,202 @@
+## TPM module with "Low Pin Count Protocol" build on SoC
+
+### Introduction
+
+The main goal of this document is showing the current state of development of TPM
+module with "LPC Protocol" based on SoC (System on Chip), and also documenting
+work carried out so far.
+
+## Motivation for doing this project
+
+We stiil see the necessity of making TPM module with "Low Pin Count" protocol.
+There is still many computers on the market (especcialy servers) which have
+mother-boards supporting LPC protocol.
+We considered possible hardware platforms in the past in this document:
+
+["Target TPM hardware interfaces"](https://lpntpm.lpnplant.io/hardware_interfaces/#low-pin-count-interface)
```

Comment:
@mgabryelski1 No need to add two extra newlines (before and after) the link. YOu should naturally place it in the text. For example: `We [have considered](hardware_interfaces.md) possible hardware platforms in the past.`

Reply:
You can (should) also use .md files as links, not the full links to the rendered documentation. It will automatically be converted to the proper links by the static content generator.